hypha/apply/activity/templates/activity/include/listing_base.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n activity_tags bleach_tags markdown_tags submission_tags apply_tags %}
-<div class="feed__item feed__item--{{ activity.type }}">
+<div class="feed__item feed__item--{{ activity.type }}" id="communications#{{ activity.id }}">
```

Comment:
Not sure about this approach here "#communication#{id}". 
Two hashes don't seem good practice but it is working fine for scrolling down to the comment.
Single #{id} was having issues with reloading.